starter-code/index.html

```diff
@@ -0,0 +1,246 @@
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+              <button type="button" class="btn btn-outline-light">READ MORE</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./img/slider-2.jpg" class="d-block w-100 myGrayOverlay" alt="...">
+            <div class="carousel-caption d-none d-md-block">
+              <h1>FALAURIS SOLLICITUDIN QUIS DOLOR VENENATIS FACILISIS</h1>
+              <p>Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam! Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non ut magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum bibendum […]</p>
+              <button type="button" class="btn btn-outline-light">READ MORE</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./img/slider-3.jpg" class="d-block w-100 myGrayOverlay" alt="...">
+            <div class="carousel-caption d-none d-md-block">
+              <h1>SUSPENDISSE ORNARE, FELIS FEUGIAT SUSCIPIT PHARETRA</h1>
+              <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra, lorem est viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+              <button type="button" class="btn btn-outline-light">READ MORE</button>
+            </div>
+          </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="sr-only">Previous</span>
+        </a>
+        <a class="carousel-control-next" href="#carouselExampleCaptions" role="button" data-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="sr-only">Next</span>
+        </a>
+    </div>
+
+    <div class="bg-danger">
+        <section class="container">
+            <!-- THREE ICONS -->
+            <div class="row text-center py-4">
+                <div class="col-md-4 text-white">
+                    <i class="fas fa-glass-martini my-5 h1"></i>
```

Comment:
To get a circle around your <i> element, you can nest it inside a <span> then apply these classes "border border-light rounded-circle p-5"